report-viewer/src/typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.json" {
```

Comment:
Is this hack really required? A quick google search showed different approaches (like [here](https://stackoverflow.com/questions/46991237/how-to-import-json-file-into-a-typescript-file)) that seam at little less hacky

Reply:
You are right, i try another way and it works too. Just add  ` "resolveJsonModule": true,` in `tsconfig`. This way is much better.